src/app/pages/applications/applications.component.ts

```diff
@@ -78,12 +72,12 @@ export class ApplicationsComponent implements OnInit, AfterViewInit {
         this.filterString = evt.data.filter;
       }
 
-      if (evt.data.event_control == 'catalogs') {
+      if (evt.data.event_control == 'catalog') {
```

Comment:
I am not sure why this is changed to 'Catalog' from 'Catalogs'.
Also I think we may remove this one as we overwrite 'selectedCatalogOptions' later in several places.
Actually we never use the value assigned here.

Reply:
Good catch! :)

Reply:
fixed, removed and tested :)